net/ipproto/ipproto.go

```diff
@@ -1,28 +1,32 @@
-// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
Put this in types/ipproto rather than net?

Reply:
Done.